### PR TITLE
[script.module.clouddrive.common] 1.1.5

### PR DESCRIPTION
**Pull request by cguZZman**
### Description
- Login app rename
- Support for images by extension only when the cloud drive doesn't provide image information. 
- Bug fixes

### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [script.foo.bar] v1.0.0

### CODE REVIEW

**Comment by cguZZman** [comment]
Just created in the meantime you guys reach a final decision about the clouddrive module and addons.

**Comment by cguZZman** [comment]
Hello Kodi Team, I just updated the module. Hope you can give me some good news and merge it.

**Comment by cguZZman** [comment]
I also updated https://github.com/xbmc/repo-plugins/pull/1534 and https://github.com/xbmc/repo-plugins/pull/1535 which depends of this pull.
Please let me know,
Thanks

**Comment by cguZZman** [comment]
Can you guys merge the plugins in the meantime you reach a decision? That way the users using the addon can have an updated version with less or no errors.

**Comment by Lunatixz** [comment]
No, it was marked invalid until the team can reach a consensus...

**Comment by Lunatixz** [comment]
@cguZZman Can you please add a disclaimer https://kodi.wiki/view/Addon.xml#.3Cdisclaimer.3E
to all plugins, scripts that use this common module and please include a note within each individual project linking to a raw copy of your heroku code. Thanks

**Comment by cguZZman** [comment]
Working on it...

**Comment by cguZZman** [comment]
@Lunatixz Changes completed.

File: script.module.clouddrive.common/clouddrive/common/ui/addon.py [hunk 1]
```diff
@@ -72,26 +70,33 @@ class CloudDriveAddon(RemoteProcessCallable):
     _export_progress_dialog_bg = None
     _system_monitor = None
     _video_file_extensions = ['mkv', 'mp4', 'avi', 'iso', 'nut', 'ogg', 'vivo', 'pva', 'nuv', 'nsv', 'nsa', 'fli', 'flc', 'wtv', 'flv']
```
Comment (Lunatixz) [inline script.module.clouddrive.common/clouddrive/common/ui/addon.py]: FYI If you are looking for Kodi supported formats; I like to use this:
```
MUSIC_EXTS         = (xbmc.getSupportedMedia('music')).split('|')  
IMAGE_EXTS         = (xbmc.getSupportedMedia('picture')).split('|')
MEDIA_EXTS         = (xbmc.getSupportedMedia('video')).split('|')
```

File: script.module.clouddrive.common/clouddrive/common/ui/addon.py [hunk 2]
```diff
@@ -353,29 +369,40 @@ def _process_items(self, items, driveid):
                 if 'thumbnail' in item:
                     list_item.setIconImage(item['thumbnail'])
                     list_item.setThumbnailImage(item['thumbnail'])
```
Comment (Lunatixz) [inline script.module.clouddrive.common/clouddrive/common/ui/addon.py]: `setThumbnailImage, setIconImage` is deprecated; please use setArt()

https://codedocs.xyz/xbmc/xbmc/group__python__xbmcgui__listitem.html#gad3f9b9befa5f3d2f4683f9957264dbbe

File: script.module.clouddrive.common/clouddrive/common/service/source.py [hunk 3]
```diff
@@ -91,148 +100,271 @@ def get_size(self, size):
     
     def get_cloud_drive_addons(self):
```
Comment (Lunatixz) [inline script.module.clouddrive.common/clouddrive/common/service/source.py]: What is the need/use of this function? THX

File: script.module.clouddrive.common/clouddrive/common/ui/addon.py (hunk 1 above)
Comment (cguZZman) [reply script.module.clouddrive.common/clouddrive/common/ui/addon.py]: That is great! I'll add it right away.

File: script.module.clouddrive.common/clouddrive/common/ui/addon.py (hunk 2 above)
Comment (cguZZman) [reply script.module.clouddrive.common/clouddrive/common/ui/addon.py]: Will be changed

File: script.module.clouddrive.common/clouddrive/common/service/source.py (hunk 3 above)
Comment (cguZZman) [reply script.module.clouddrive.common/clouddrive/common/service/source.py]: I have a source mode in the addons. 
You can add your drive as a source in Kodi by pointing to http://localhost:8585/source and you will be able to simulate a web directory.
The first page in that navigation is the list of addons supporting this feature, which I discover by checking the list of addons dependant on this common module.

**Comment by cguZZman** [comment]
Changes done

File: script.module.clouddrive.common/clouddrive/common/service/source.py (hunk 3 above)
Comment (Lunatixz) [reply script.module.clouddrive.common/clouddrive/common/service/source.py]: This will require further review :)

Comment (cguZZman) [reply script.module.clouddrive.common/clouddrive/common/service/source.py]: No worries. Let me know if more explanation is necessary.

**Comment by cguZZman** [comment]
Just added some improvements to the cache class and removed some debug logs statements.

**Pull request merged**